plugin-management-cli/src/main/java/io/jenkins/tools/pluginmanager/cli/CliOptions.java

```diff
@@ -9,31 +9,57 @@
 import org.kohsuke.args4j.spi.StringArrayOptionHandler;
 
 
-
 public class CliOptions {
     //path must include plugins.txt
     @Option(name = "-pluginTxtPath", usage = "Path to plugins.txt")
     public File pluginTxt;
 
+
     @Option(name = "-pluginDirPath", usage = "Directory in which to install plugins")
     public File pluginDir;
 
-    @Option(name = "-plugins", usage = "List of plugins to install, separated by a space", handler = StringArrayOptionHandler.class)
-    public String[] plugins;
+
+    @Option(name = "-plugins", usage = "List of plugins to install, separated by a space",
+            handler = StringArrayOptionHandler.class)
+    public String[] plugins = new String[0];
+
 
     @Option(name = "-war", usage = "Path to Jenkins war file")
     public String jenkinsWarFile;
 
-    @Option(name = "-viewSecurityWarnings", usage = "Set to true to show specified plugins that have security warnings", handler = BooleanOptionHandler.class)
+    @Option(name = "-viewSecurityWarnings", usage = "Set to true to show specified plugins that have security warnings",
+            handler = BooleanOptionHandler.class)
     public boolean showWarnings;
 
-    @Option(name = "-viewAllSecurityWarnings", usage = "Set to true to show all plugins that have security warnings", handler = BooleanOptionHandler.class)
+    @Option(name = "-viewAllSecurityWarnings", usage = "Set to true to show all plugins that have security warnings",
+            handler = BooleanOptionHandler.class)
     public boolean showAllWarnings;
 
+
+    @Option(name = "-JENKINS_UC",
+            usage = "Sets main update center; will override environment variable. If not set via CLI option or " +
+                    "environment variable, will default to https://updates.jenkins.io")
+    public String jenkinsUc;
+
+
+    @Option(name = "-JENKINS_UC_EXPERIMENTAL",
```

Comment:
better to be more verbose, and more standard design, I've never seen an upper case argument before
```suggestion
    @Option(name = "-jenkins-experimental-update-center",
```